compiler/codegen/ELFObjectFileGenerator.hpp

```diff
@@ -0,0 +1,145 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2014, 2017
+ *
+ *  This program and the accompanying materials are made available
+ *  under the terms of the Eclipse Public License v1.0 and
+ *  Apache License v2.0 which accompanies this distribution.
+ *
+ *      The Eclipse Public License is available at
+ *      http://www.eclipse.org/legal/epl-v10.html
+ *
+ *      The Apache License v2.0 is available at
+ *      http://www.opensource.org/licenses/apache2.0.php
+ *
+ * Contributors:
+ *    Multiple authors (IBM Corp.) - initial implementation and documentation
+ *******************************************************************************/
+
+#ifndef ELFFILEGENERATOR_HPP
+#define ELFFILEGENERATOR_HPP
+
+#if defined(LINUX)
+
+#include <elf.h>
```

Comment:
Will `elf.h` always be present on Linux systems Eclipse OMR will be built on or is there a separate package to install?